lib/tasks/create_application.rake

```diff
@@ -7,24 +7,24 @@ namespace :auth do
 
     puts "What's the application's name?"
     name = STDIN.gets.chomp
-
     application = Doorkeeper::Application.new(name: name)
 
-    suppliers = Supplier.all.each_with_object({}) { |supplier, accumulator|
-      accumulator[supplier.name.downcase] = supplier.id
-    }.merge('none' => nil)
 
-    puts "Which application owner should your tokens be associated with? #{suppliers.keys}"
-    supplier = STDIN.gets.chomp
-    supplier_id = suppliers[supplier]
+    keys = Supplier.pluck(:key).sort + ['none']
+    puts "Which application owner should your tokens be associated with? #{keys}"
+
+    supplier_key = STDIN.gets.chomp
+    abort("Error: unknown supplier, quitting.") unless keys.include?(supplier_key)
 
-    application.owner_id = supplier_id if supplier_id.present?
+    supplier = supplier_key == 'none' ? nil : Supplier.find_by(key: supplier_key)
```

Comment:
for clarity
```suggestion
    supplier = (supplier_key == 'none' ? nil) : Supplier.find_by(key: supplier_key)
```

Reply:
Looks like bracket is in the wrong position in the suggested change ☝️

Reply:
Sorry @alexdesi, RuboCop says no:
```
lib/tasks/create_application.rake:19:16: C: [Corrected] Style/TernaryParentheses: Omit parentheses for ternary conditions.
    supplier = (supplier_key == 'none') ? nil : Supplier.find_by(key: supplier_key)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```